### PR TITLE
Email verification

### PR DESCRIPTION
**Pull request by chamod-gamage**
Created async service function `handleVerifyEmail` that takes a actionCode (Firebase generated oobCode in the URL) as a parameter and verifies user in Firebase.
**Also updated Frontend ENV variables to include REACT_APP**

### CODE REVIEW

**Comment by chamod-gamage** [comment]
``` const [confirmationErrors, setConfirmationErrors] = useState("");
  useEffect(() => {
    async function verifyEmail() {
      const URLParams = new URLSearchParams(window.location.href);
      const actionCode = URLParams.get("oobCode");
      if (actionCode) {
        handleVerifyEmail(actionCode)
          .then((err: string) => {
            setConfirmationErrors(err);
          })
          .catch((err: string) => {
            setConfirmationErrors(err);
          });
        console.log(confirmationErrors);
      }
    }
    verifyEmail();
  }, []); 
```

@jlight99 You can include this code in the frontend page you make at `/verify-email` and render the Email Confirmed modal conditionally based on confirmationErrors

File: client/src/services/auth.ts [hunk 1]
```diff
@@ -53,3 +53,19 @@ export const createNewAccount = async (
 
   return errors;
 };
+
+export const handleVerifyEmail = async (
+  actionCode: string
+): Promise<string> => {
+  let error = "";
+  error = await firebase
```
Comment (meganniu) [inline client/src/services/auth.ts]: Nitpick:
On line 60, `error` is initialized as a `string`.
On line 61, `error` is assigned a value with type `Promise<string>`

my suggestion is to remove line 60 and just initialize `error` as a `Promise<string>`

**Review by meganniu — APPROVED** [review]
LGTM!!

**Review by jlight99 — APPROVED** [review]
lgtm!

**Comment by jlight99** [comment]
> ```
>  useEffect(() => {
>    async function verifyEmail() {
>      const URLParams = new URLSearchParams(window.location.href);
>      const actionCode = URLParams.get("oobCode");
>      if (actionCode) {
>        handleVerifyEmail(actionCode)
>          .then((err: string) => {
>            setConfirmationErrors(err);
>          })
>          .catch((err: string) => {
>            setConfirmationErrors(err);
>          });
>        console.log(confirmationErrors);
>      }
>    }
>    verifyEmail();
>  }, []); 
> ```
> 
> @jlight99 You can include this code in the frontend page you make at `/verify-email` and render the Email Confirmed modal conditionally based on confirmationErrors

tyty!

**Pull request merged**